curriculum/challenges/english/03-front-end-development-libraries/front-end-development-libraries-projects/build-a-25-5-clock.md

```diff
@@ -8,9 +8,11 @@ dashedName: build-a-25--5-clock
 
 # --description--
 
-**Objective:** Build a [CodePen.io](https://codepen.io) app that is functionally similar to this: <https://codepen.io/freeCodeCamp/full/XpKrrW>.
+**Objective:** Build an app that is functionally similar to this: <https://codepen.io/freeCodeCamp/full/XpKrrW>.
 
-Fulfill the below [user stories](https://en.wikipedia.org/wiki/User_story) and get all of the tests to pass. Give it your own personal style.
+Fulfill the below user stories. Use whichever libraries or APIs you need. Give it your own personal style.
```

Comment:
I would keep the "and get all of the tests to pass" on all the projects that have automated tests.

```suggestion
Fulfill the below user stories and get all of the tests to pass. Use whichever libraries or APIs you need. Give it your own personal style.
```

Reply:
Sure I am on it